packages/e2e-test-utils/src/create-user.js

```diff
@@ -10,14 +10,25 @@ import { switchUserToAdmin } from './switch-user-to-admin';
 import { switchUserToTest } from './switch-user-to-test';
 import { visitAdminPage } from './visit-admin-page';
 
+/**
+ * @typedef {'Subscriber' | 'Contributor' | 'Author' | 'Editor' | 'Administrator'} Role
+ */
+
 /**
  * Create a new user account.
  *
- * @param {string}  username  User name.
- * @param {string?} firstName First name.
- * @param {string?} lastName  Larst name.
+ * @param {string}  username           User name.
+ * @param {Object?} object             Optional Settings for the new user account.
+ * @param {string}  [object.firstName] First name.
+ * @param {string}  [object.lastName]  Last name.
+ * @param {Role}    [object.role]      Role.
```

Comment:
Should we add a comment here that if no role is provided, the default will be used, and state which it is?